src/api/app/controllers/comments_controller.rb

```diff
@@ -2,7 +2,7 @@ class CommentsController < ApplicationController
   before_action :find_obj, only: [:index, :create]
 
   def index
-    @comments = @obj.comments.order(:id)
+    @comments = @obj.comments.preload(:user).order(:id)
```

Comment:
any significative change in performance if you use `eager_load` instead? :thinking: